### PR TITLE
Fix hoisting of statics

### PR DESCRIPTION
**Pull request by Andreyco**

### CODE REVIEW

**Comment by Andreyco** [comment]
Fixes #108

File: src/hoistStatics.tsx [hunk 1]
```diff
@@ -11,6 +11,16 @@ const REACT_STATICS: any = {
   type: true,
 };
 
+var KNOWN_STATICS: any = {
```
Comment (jaredpalmer) [inline src/hoistStatics.tsx]: nit: use `let` here

Comment (Andreyco) [inline src/hoistStatics.tsx]: I think `const` will serve even better here.

Comment (jaredpalmer) [inline src/hoistStatics.tsx]: Yes, my b

**Comment by jaredpalmer** [comment]
LGTM

**Comment by Andreyco** [comment]
1450460008joys-of-block-scoping-with-es602-make-let-not-var

**Pull request merged**